.gitignore

```diff
@@ -1,4 +1,7 @@
 node_modules
```

Comment:
Happy for you to tidy this up and add it to your PR. By tidy up, maybe just remove `nbproject` as that looks unrelated.